src/string.rs

```diff
@@ -223,6 +224,24 @@ fn not_whitespace_except_line_feed(g: &str) -> bool {
 /// FIXME(issue#3281): We must follow UAX#14 algorithm instead of this.
 fn break_string(max_width: usize, trim_end: bool, line_end: &str, input: &[&str]) -> SnippetState {
     let break_at = |index /* grapheme at index is included */| {
+        // Ensure break is not after an escape '\' as it will "escape" the `\` that is added
+        // for concatenating the two parts of the broken line.
```

Comment:
I think this rephrasing could help explain that we're trying not to escape any line continuation characters that we add when we break the string.

```suggestion
        // Ensure break is not after an escape '\' as it will "escape" the line continuation `\`
        // that is added for concatenating the two parts of the broken line.
```

Reply:
Comment enhanced in line with the requested change.